src/redist/targets/BundledTemplates.targets

```diff
@@ -44,8 +44,8 @@
     <Bundled30Templates Include="Microsoft.Dotnet.Wpf.ProjectTemplates" PackageVersion="$(MicrosoftDotnetWpfProjectTemplates30PackageVersion)" />
     <Bundled30Templates Include="Microsoft.Dotnet.WinForms.ProjectTemplates" PackageVersion="$(MicrosoftDotnetWinFormsProjectTemplates30PackageVersion)" />
     <Bundled30Templates Include="Microsoft.DotNet.Web.ItemTemplates" PackageVersion="$(AspNetCorePackageVersionFor30Templates)" />
-    <Bundled30Templates Include="Microsoft.DotNet.Web.ProjectTemplates.3.0" PackageVersion="$(AspNetCorePackageVersionFor30Templates)" />
-    <Bundled30Templates Include="Microsoft.DotNet.Web.Spa.ProjectTemplates.3.0" PackageVersion="$(AspNetCorePackageVersionFor30Templates)" />
+    <Bundled30Templates Include="Microsoft.DotNet.Web.ProjectTemplates.5.0" PackageVersion="$(AspNetCorePackageVersionFor30Templates)" />
+    <Bundled30Templates Include="Microsoft.DotNet.Web.Spa.ProjectTemplates.5.0" PackageVersion="$(AspNetCorePackageVersionFor30Templates)" />
```

Comment:
This is a hack. I will file an issue to fix up the template handling.

Reply:
Slightly separate, but I think we should stop using `<FooPackageVersion>$(BarPackageVersion)</FooPackageVersion>` in Version.props.

1. It can be confusing
2. The versions are machine-edited so there's no value in writing fewer versions
3. It assumes that repos won't ship a subset of packages to us in servicing.

For example, we are assuming that targeting pack version is runtime pack version right now, which is WRONG. We will want to stay on 3.0.0 targeting pack unless tghere's a bug. I think this should be fixed up in release/3.0.1xx as part of servicing readiness. I will file an issue.

Reply:
https://github.com/dotnet/core-sdk/blob/be0281b5dc0860d207842d5b6e3755bd9503f569/eng/Versions.props#L53